src/prop/prop_engine.cpp

```diff
@@ -41,6 +41,8 @@
 #include "util/resource_manager.h"
 #include "util/result.h"
 
+using namespace std;
```

Comment:
why this addition? I'd rather we were removing these.

Reply:
I removed one from a header file, but didn't feel like immediately fixing the whole source file.

Reply:
I see. Yeah, in that case better to leave it like this.

Reply:
As long as it gets removed eventually. Else, I'd rather have it done in this PR since you already touched it.

Reply:
Okay, I've removed it.